packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -139,7 +139,7 @@ function fireUpdate<T extends Item>(
 }
 
 function defaultRenderInput(props: ComponentProps<typeof Input>) {
-  return <Input {...props} />;
+  return <Input data-1p-ignore {...props} />;
```

Comment:
Could you add a comment above just explaining what the class is and why it's there please?

```suggestion
  // data-1p-ignore disables 1Password autofill behaviour
  return <Input data-1p-ignore {...props} />;
```